src/pages/ChatPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import './ChatPage.css';
+
+const ChatPage = ({location, history}) => {
+    const name = location.state;
+    const [message, setMessage] = useState([]);
+    const userText=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
+    const [value, setValue] = useState('');
+    const onChange = (event) => {
+        setValue(event.target.value);
+    };
+    
+    const sendFunc = () => {
+        if(value === '')  return;
+        let tok = false;
+        let idx = 0;
+        if(Math.floor(Math.random() * 2)){
```

Comment:
```suggestion
        if(Math.random() < 0.5){
```
50% 확률이면 이게 더 깔끔한 것 같아요